### PR TITLE
feat: add default_socket_timeout configuration

### PR DESCRIPTION
**Pull request by voroyam**

### CODE REVIEW

File: latest/overlay/etc/templates/owncloud.ini.tmpl [hunk 1]
```diff
@@ -8,6 +8,8 @@ post_max_size = "{{ .Env.OWNCLOUD_MAX_UPLOAD }}"
 max_execution_time = {{ .Env.OWNCLOUD_MAX_EXECUTION_TIME }}
 max_input_time = {{ .Env.OWNCLOUD_MAX_INPUT_TIME }}
 
+default_socket_timeout = "${{ .Env.OWNCLOUD_DEFAULT_SOCKET_TIMEOUT }}"
```
Comment (xoxys) [inline latest/overlay/etc/templates/owncloud.ini.tmpl:11]:
```suggestion
default_socket_timeout = {{ .Env.OWNCLOUD_DEFAULT_SOCKET_TIMEOUT }}
```

File: v20.04/overlay/etc/templates/owncloud.ini.tmpl [hunk 2]
```diff
@@ -8,6 +8,8 @@ post_max_size = "{{ .Env.OWNCLOUD_MAX_UPLOAD }}"
 max_execution_time = {{ .Env.OWNCLOUD_MAX_EXECUTION_TIME }}
 max_input_time = {{ .Env.OWNCLOUD_MAX_INPUT_TIME }}
 
+default_socket_timeout = "${{ .Env.OWNCLOUD_DEFAULT_SOCKET_TIMEOUT }}"
```
Comment (xoxys) [inline v20.04/overlay/etc/templates/owncloud.ini.tmpl:11]:
```suggestion
default_socket_timeout = {{ .Env.OWNCLOUD_DEFAULT_SOCKET_TIMEOUT }}
```

**Review by xoxys — CHANGES_REQUESTED** [review]
Please also add it with the default value to https://github.com/owncloud-docker/base/blob/master/v20.04/overlay/etc/php/7.4/mods-available/owncloud.ini (and the latest directory)

**Review by xoxys — APPROVED** [review]

**Comment by phil-davis** [comment]
Note: https://github.com/owncloud-docker/base/commit/9df7aa89b8811702dc99d555d1e0cf07da7d8186
`default_socket_timeoute` had been set to 3600, and was deleted in that commit.
PR #225 
Maybe that was an accidental deletion in that PR?

**Comment by xoxys** [comment]
I had removed it back in the days because it was not set consistently on all images. Setting it to `60` seconds is the php default and should be fine. What do you think?

**Comment by voroyam** [comment]
merge?

**Comment by xoxys** [comment]
@phil-davis Feel free to still add any objections.

**Comment by phil-davis** [comment]
Looks OK to me.

**Pull request merged**